### DISCUSSION TITLE
Generate test HIDS alert for Wazuh

### DISCUSSION THREAD

**Discussion by jackface0** [Security-Onion-Solutions/securityonion#7790] [Unsupported Versions]
I have installed the agents for winlogbeat (with sysmon), osquery and Wazuh.

I can see the logs coming in from sysmon and I can see my test hosts in Fleet. I'm wondering if there is a way to generate a test HIDS event on the host so that an alert shows up in the alerts page in security onion web interface. I just want to confirm its working

![image](https://user-images.githubusercontent.com/89797177/163361987-eda1820b-2270-4be3-bb63-04e9d3ef5d79.png)

**Comment by dougburks** [comment]
Sounds like you want a way to generate a Wazuh HIDS alert for a Windows system. Take a look at the Windows rules at `/nsm/wazuh/ruleset/rules/*win*`.
